packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -137,7 +133,7 @@ export default function FeedItemComponent({
           ref={inViewRef}
           post={{
             ...item.post,
-            createdAt: displayPublicationDate && item.post.createdAt,
+            createdAt: item.post.createdAt,
```

Comment:
I think we can just pass `item.post` here and no need for destructuring since there is no modification with the data anymore.